app/init.php

```diff
@@ -3,9 +3,17 @@
 $dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
 $dotenv->load();
 
+$loader = new \Twig\Loader\FilesystemLoader(__DIR__ . '/templates');
+$twigConfig = [];
```

Comment:
In session, I guided y'all wrong. This pattern is better:
- first, set up `$twigConfig` to hold configuration variables for the template engine. (it's empty)